### DISCUSSION TITLE
Token

### DISCUSSION THREAD

**Discussion by TzaraDuchamp** [MartinPham/FindMySync#7] [Q&A]
How can you create a token for this? I followed the home assistant guide and also tried web socket in Chrome, no luck. The app also keeps changing the endpoint ip in the url to /homeassistant.local. Can you show us in a video or with image how you got this to work? 

The sensors do show up, but they show as not_home and no other information. Is this because the known_devices.yaml is deprecated? 

Can we remove the sensors from Home Assistant, when I rename them the old ones seem to stay.

**Comment by mshoe007** [comment]
I found when entering the URL, I have to press <ENTER> instead of <TAB> to go to the next field. If I press <TAB>, then it does not appear that the URL I enter gets saved.

Same with the token. 

> How can you create a token for this? 

In the home assistant app or web page, click on your user name at the lower left. This should open your profile page.

Scroll all of the way to the bottom, and the last heading is **Long-Lived Access Tokens**

Click on **CREATE TOKEN**

Enter a name for the token. I used FindMyApp  

Press OK. The token appears in a dialog box as a string of about 200 characters. Copy and paste that string someplace safe. You can't recover it from Home Assistant. After saving the string somewhere, paste it into the BEARER field in the FindMyApp and press ENTER (not <TAB> or any other usual Mac key that works in every other app for filling in fields).
